virttest/iscsi.py

```diff
@@ -843,7 +843,10 @@ def create_iSCSI(params, root_dir=data_dir.get_tmp_dir()):
             raise exceptions.TestError("Failed to install iscsi initiator"
                                        " packages")
         # Install linux iscsi target software targetcli
-        iscsi_package = ["targetcli"]
+        if distro.detect().version == '18' and distro.detect().name == 'Ubuntu':
+            iscsi_package = ["targetcli-fb"]
+        else:
+            iscsi_package = ["targetcli"]
```

Comment:
have ```try...
block, ```
to handle it generic for all distros, future some other distro versions might also have this naming.
...

Reply:
if i try to install targetcli on 18.04 version it will not error out because wait_for_login(), and some times guest got crashed.

Reply:
The better way for this would to implement a API in utils_package that would give you the package name by querying irrespective of distro, like `yum whatprovides targetcli` in this case for RHEL based package manager and similar in Ubuntu